functests/09-test-base-image-namespace_rbac.sh

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash
+
+SCRIPTPATH=$( dirname $(readlink -f $0) )
+source ${SCRIPTPATH}/testlib.sh
+
+RET=1
+TEST_NS="kubevirt-os-images"
+EDIT_ROLE="os-images.kubevirt.io:edit"
+VIEW_ROLE="os-images.kubevirt.io:view"
+
+function find_namespace {
+        # Find if the Base Image Namespace is created
+	local status=1
+        for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find namespace ${TEST_NS}"
+                oc get ns | grep  ${TEST_NS} > /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found Namespace in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+        echo $status
+}
+
+function find_editclusterrole {
+        # Find if the Edit Clusterrole is created
+        
+	local status=1
+	for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find clusterrole ${EDIT_ROLE}"
+                oc get clusterrole | grep ${EDIT_ROLE} >  /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found clusterrole in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+	echo $status
+}
+
+function find_viewrole {
+        # Find if the View Role is created
+
+	local status=1
+	for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find role ${VIEW_ROLE}"
+                oc get role -n ${TEST_NS} | grep ${VIEW_ROLE} >  /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found role in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+	echo $status
+}
+
+function find_viewrolebinding {
+        # Find if the View Rolebinding is created
+
+	local status=1
+	for idx in $( seq 1 40); do
+	        (( ${V} >= 1 )) && echo "Attempt ${idx} to find rolebinding ${VIEW_ROLE}"
+                oc get rolebinding -n ${TEST_NS} | grep ${VIEW_ROLE} >  /dev/null
+                if [ $? == 0 ]; then
+                        status=0
+	                (( ${V} >= 1 )) && echo "Found rolebinding in Attempt ${idx}"
+                        break
+                fi
+	        sleep 3s
+        done
+	echo $status
+}
+
+oc create -n ${SSP_OP_POD_NAMESPACE} -f "${SCRIPTPATH}/common-templates-unversioned-cr.yaml" || exit 2
+# TODO: SSP-operator needs to improve its feedback mechanism
+sleep 21s
+
+# Check if the namespace is created
+echo "CNV-4494: SSP Base Image OS namespace is created"
```

Comment:
Are these proper test ids? 
I am used to see test ids like this: https://github.com/omeryahud/ssp-operator/pull/8/files

Reply:
@rnetser

Reply:
@omeryahud basically you are right; test IDs should be in https://github.com/omeryahud/ssp-operator/pull/8/files format when written in go.
Here they are written in bash, so I asked Shweta to check how other bash tests are echoing the id (even it this is not useful now for tests result; the code should direct to the polarion IDs so when will be handy when moving to go)